Settings.c

```diff
@@ -423,6 +428,9 @@ Settings* Settings_new(int initialCpuCount) {
       this->showMergedCommand = false;
       this->headerMargin = true;
    }
+
+   settings = this;
+
```

Comment:
if we were to store this globally, there should be some proper reuse strategy, as done in this way may cause hard-to-debug issues. Routines should not have non-obvious globally visible side-effects.